### PR TITLE
Upgrade rails to 5.2.3

### PR DESCRIPTION
**Pull request by elena-vi**
- Upgrade rails with `rails app:update`
- Also updated bundler, which required an addition to the CircleCI config file as CircleCI apparentl doesn't come with bundler 2.

### CODE REVIEW

File: config/initializers/new_framework_defaults_5_2.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Be sure to restart your server when you modify this file.
```
Comment (tommotaylor) [inline config/initializers/new_framework_defaults_5_2.rb]: We can delete this file

File: Gemfile.lock [hunk 2]
```diff
@@ -416,4 +419,4 @@ RUBY VERSION
    ruby 2.5.1p57
 
 BUNDLED WITH
-   1.17.2
+   1.16.6
```
Comment (tommotaylor) [inline Gemfile.lock]: Doesn't seem right

File: Gemfile [hunk 3]
```diff
@@ -17,6 +17,9 @@ ruby '2.5.1'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'bootsnap', '>= 1.1.0', require: false
```
Comment (emileswarts) [inline Gemfile]: We need to pull this out again at some point.  Ok with that being done later for all the apps that were upgraded.

Comment (elena-vi) [reply Gemfile]: why?

**Pull request merged**